src/code/z_actor.c

```diff
@@ -1550,18 +1561,28 @@ s32 func_8002F1C4(Actor* actor, PlayState* play, f32 arg2, f32 arg3, u32 exchang
     return true;
 }
 
-s32 func_8002F298(Actor* actor, PlayState* play, f32 arg2, u32 exchangeItemId) {
-    return func_8002F1C4(actor, play, arg2, arg2, exchangeItemId);
+/**
+ * Offers a talk exchange request within an equilateral cylinder with the radius specified.
+ */
+s32 Actor_OfferTalkExchangeEquiCylinder(Actor* actor, PlayState* play, f32 radius, u32 exchangeItemId) {
+    return Actor_OfferTalkExchange(actor, play, radius, radius, exchangeItemId);
 }
 
-s32 func_8002F2CC(Actor* actor, PlayState* play, f32 arg2) {
-    return func_8002F298(actor, play, arg2, EXCH_ITEM_NONE);
+/**
+ * Offers a talk request within an equilateral cylinder with the radius specified.
+ */
+s32 Actor_OfferTalk(Actor* actor, PlayState* play, f32 radius) {
+    return Actor_OfferTalkExchangeEquiCylinder(actor, play, radius, EXCH_ITEM_NONE);
 }
 
-s32 func_8002F2F4(Actor* actor, PlayState* play) {
-    f32 var1 = 50.0f + actor->colChkInfo.cylRadius;
+/**
+ * Offers a talk request within an equilateral cylinder whose radius is determined by the actor's collision check
+ * cylinder's radius.
+ */
+s32 Actor_OfferTalkNearColChkInfoCylinder(Actor* actor, PlayState* play) {
```

Comment:
I think this would be fine as
```suggestion
s32 Actor_OfferTalkNearby(Actor* actor, PlayState* play) {
```
or if there's room to forget it depends on the colChkInfo
```suggestion
s32 Actor_OfferTalkNearColChk(Actor* actor, PlayState* play) {
```
but I suggest waiting for more opinions

Reply:
Similar conversation in MM https://github.com/zeldaret/mm/pull/1312#discussion_r1250810347

Given the strange nature of this cylinder i dont think theres any separating it. Maybe the name can be better one day if we give a better name to the cylinder in question. But I think it should be named after it explicitly

Reply:
I see keyboards were already smashed over this

this is fine then, and it's only used 20ish times. It unfortunately jumped out to me because the first actor in the diff, `z_en_a_keep`, uses it lol